<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export</class>
 <widget class="QDialog" name="Export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames Per Second:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExportFPSSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>16X16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32X32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64X64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128X128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256X256</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>512X512</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="actualSizeRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Size</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="otherSizeRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Other Size</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Export</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Export</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
